tcpip.cc

```diff
@@ -839,7 +839,7 @@ int send_tcp_raw_decoys(int sd, const struct eth_nfo *eth,
 
   for (decoy = 0; decoy < o.numdecoys; decoy++)
     if (send_tcp_raw(sd, eth,
-                     &o.decoys[decoy], victim,
+                     &((struct sockaddr_in *)&o.decoys[decoy])->sin_addr, victim,
```

Comment:
Again no extra work for IPv6 just changed this to make it compatible with new datatype of `o.decoys[]`